test/test_metric/test_metrics.py

```diff
@@ -295,6 +295,63 @@ def test_threshold_scorer_multilabel(self):
         score = scorer(y_true, y_pred)
         self.assertAlmostEqual(score, -1.0)
 
+    def test_threshold_scorer_multi_class(self):
+        y_true = np.array([[0, 1], [1, 0], [1, 0], [0, 1]])
```

Comment:
Hey, I'm afraid this is not testing what it's supposed to test. Right now, this tests for binary classification, which roc_auc can handle anyways. Could you please extend this to a multiclass problem?